src/vs/editor/common/config/editorOptions.ts

```diff
@@ -3229,14 +3228,18 @@ class EditorActionBar extends BaseEditorOption<EditorOption.actionBar, IEditorAc
 			'actionBar',
 			defaults,
 			{
-				'editor.actionBar.enabled': {
-					type: 'boolean',
-					default: defaults.enabled,
-					description: nls.localize(
-						'actionBar.enabled',
-						"Controls whether the action bar is shown."
+				'editor.actionBar.hiddenForLanguages': {
+					'type': 'array',
+					'items': {
+						'type': 'string'
+					},
+					'markdownDescription': nls.localize(
+						'editor.actionbar.hiddenForLanguages',
+						"Configures the languages for which the action bar will be hidden. For example, `python` will hide the action bar for Python files, `r` will hide the action bar for R files, and `plaintext` will hide the action bar for plain text files. Use `*` to hide the action bar for all languages. The default is `plaintext`.",
```

Comment:
```suggestion
						"Configures the languages for which the action bar will be hidden. For example, `python` will hide the action bar for Python files, `r` will hide the action bar for R files, and `plaintext` will hide the action bar for plain text files. Use `*` to hide the action bar for all languages.",
```

It doesn't appear that this is really the default? In my testing plaintext files get the action bar by default, and the action bar is removed if I add a new `plaintext` entry to this list.

Reply:
I fixed this already!